src/locationListPage-state/actions/listAction.js

```diff
@@ -0,0 +1,10 @@
+import * as actionTypes from "./actionTypes";
+
+export const addLocation = (id, lat, lng) => {
+  return {
+    type: actionTypes.ADD_LOCATION,
```

Comment:
no checks for the values, either some checks should be there or some fallback default values